kafka-ui-react-app/src/components/Connect/Edit/__tests__/Edit.spec.tsx

```diff
@@ -85,10 +85,8 @@ describe('Edit', () => {
 
     it('calls updateConfig on form submit', async () => {
       const updateConfig = jest.fn();
-      const wrapper = mount(setupWrapper({ updateConfig }));
-      await act(async () => {
-        wrapper.find('form').simulate('submit');
-      });
+      renderComponent({ updateConfig });
+      await waitFor(() => fireEvent.submit(screen.getByRole('form')));
```

Comment:
```js
await waitFor(() => fireEvent.submit(screen.getByRole('form')));
```
Is use of `fireEvent.submit` - desired approach?

I have thought about case where someone deletes `Submit` button and this test won't fail

I'm not sure about this
```js
userEvent.click(screen.getByText('Submit'));
```

Reply:
got a lot of problems with submitting react hook form because of connector
this is the only approach which works